mudlet-mapper.xml

```diff
@@ -5678,6 +5680,28 @@ function mmp.addWingsAchaea()
       mmp.tempSpecialExit(54632, &quot;Shake Harness&quot;)
     end
   end
+		
+	-- Air Elemental Aero Soar
+	  if
+			mmp.settings.soar and gmcp.Room and not table.contains(gmcp.Room.Info.details, &quot;indoors&quot;)
+  then
+    --Sarapis soar.
+    if mmp.oncontinent(getRoomArea(mmp.currentroom), &quot;Main&quot;) then
+			if gmcp.Char.Status.race == &quot;Elemental Lord&quot; then
+      	mmp.tempSpecialExit(4882, &quot;aero soar high&quot;, 10) -- duanatharan
+       	mmp.tempSpecialExit(3885, &quot;aero soar low&quot;, 10) -- duanathar
+  			mmp.tempSpecialExit(54173, &quot;aero soar stratosphere&quot;, 10) -- Stratosphere!
+			end
+		--Meropis soar
+    elseif mmp.oncontinent(getRoomArea(mmp.currentroom), &quot;Meropis&quot;) then
+			if gmcp.Char.Status.race == &quot;Elemental Lord&quot; then
```

Comment:
I would do it just a step further just in case in putting gmcp.Char.Status.class == &quot;air Elemental Lord&quot; then
only reason i say that is the just in case someone changes factions and still has that enabled that it wont try to use it if they are a different elemental lord

Reply:
Not when you use the class string instead of race.

Reply:
Oh, yeah you are right. Okay then I'm going to do that

Reply:
Done :)